### DISCUSSION TITLE
Welcome to nanogram.js Discussions!

### DISCUSSION THREAD

**Discussion by webistomin** [webistomin/nanogram.js#259] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by scc4** [comment]
Is it still working?

**Reply by webistomin** [reply]
There is no solution for the browser yet. If you need an implementation for node.js, [here's](https://github.com/webistomin/nanogram.js/issues/402#issuecomment-820542527) the solution. You can host your server and send a request directly to it.
